src/core/tracking.js

```diff
@@ -322,4 +324,62 @@ export const sendSponsoredEventBeacon = ({
   });
 };
 
+export const storeConversionInfo = ({
+  _log = log,
+  _window = typeof window !== 'undefined' ? window : {},
+  addonId,
+  data,
+}: {
+  _log?: typeof log,
+  _window?: typeof window | Object,
+  addonId: number,
+  data: string,
+}) => {
+  if (_window.sessionStorage) {
+    _window.sessionStorage.setItem(
+      SPONSORED_INSTALL_CONVERSION_INFO_KEY,
+      JSON.stringify({ addonId, data }),
```

Comment:
`JSON.stringify` could throw it seems: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/JSON/stringify

Reply:
Fixed